syntax = "proto3";

import "hq/dto/Security.proto";
import "hq/dto/TimeShare.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.response";
option java_outer_classname = "TimeShareResponseProto";
message TimeShareResponse {
	Security security = 1; //股票定义
	repeated TimeShare  timeShare = 2;
}
